<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherMainWindow</class>
 <widget class="QMainWindow" name="TeacherMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Teacher Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	border-radius:3px;
}
QDialog{
background-color:rgb(230, 230, 230);
}
QLineEdit
{
    font: 25 9pt &quot;Microsoft JhengHei&quot;;
    background:white;
    padding-left:4px ;
    padding-top:4px ;
    border-bottom-left-radius:4px;
    border-bottom-right-radius:4px;
    border: 1.2px solid rgb(189 , 189, 189);
    border-top:transparent;
    
}
QLineEdit:hover
{
    font: 25 9pt &quot;Microsoft JhengHei&quot;;
    padding-top:0px ;
    border: 1px solid rgb(21 , 131 , 221);
}
QLineEdit:focus
{
    font: 25 9pt &quot;Microsoft JhengHei&quot;;
    padding-top:0px ;
    border: 1px solid rgb(21 , 131 , 221);
}

QPushButton
{
	font: 25 9pt &quot;Microsoft JhengHei&quot;;
    color:white;
    background-color:rgb(75,164,242);
    border-radius:3px;
}

QPushButton:hover
{
	font: 25 10pt &quot;Microsoft JhengHei&quot;;
    color:white;
    background-color:rgb(44 , 137 , 255);
	border-radius:5px;
}

QPushButton:pressed
{
	font: 25 11pt &quot;Microsoft JhengHei&quot;;
	border-radius:13px;
    color:white;
    background-color:rgb(14 , 135 , 228);
    padding-left:5px;
    padding-top:5px;
}

QPushButton:default
{
	font: 25 9pt &quot;Microsoft JhengHei&quot;;
	border-radius:3px;
    color:white;
    background-color:rgb(44 , 137 , 255);
    padding-left:3px;
    padding-top:3px;
    border: 2px solid rgb(17 , 120 , 221);
}

QPushButton:!enabled
{
	font: 25 9pt &quot;Microsoft JhengHei&quot;;
    color:white;
    background-color:rgb(200 , 200 , 200);
	border-radius:5px;
}
QRadioButton
{
    font: 25 9pt &quot;Microsoft JhengHei&quot;;
}
QCheckBox
{
    font: 25 9pt &quot;Microsoft JhengHei&quot;;
}
QLabel#TopImage
{
	border-radius:90px;
}
QPushButton#btnExit{
border-image: url(:/TeacherWindow/image/timg.png);
background-color: rgb(240, 240, 240);
border-radius:0px;
}
QPushButton#btnExit:hover{
border-image: url(:/TeacherWindow/image/timg.png);
background-color: rgb(221, 0, 27);
border-radius:0px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>913</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnExit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>答题记录</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnNewProblem">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>发题</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btnBeginClass">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>上课</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="btnChat">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>聊天</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnEndClass">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>下课</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="ckbVoiceShare">
        <property name="text">
         <string>语音</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckbScreenShare">
        <property name="text">
         <string>投屏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbAttandance">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft JhengHei</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbStudent">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft JhengHei</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="currentText">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
